mitiq/ddd/tests/test_ddd.py

```diff
@@ -181,6 +181,9 @@ def test_mitigate_executor_ddd():
     mitigated_executor = mitigate_executor(serial_executor, rule=xx)
     assert np.isclose(mitigated_executor(circuit_cirq_a), ddd_value)
 
+    batched_mitigated_executor = mitigate_executor(batched_executor, rule=xx)
+    assert np.isclose(*batched_mitigated_executor([circuit_cirq_a]), ddd_value)
+
```

Comment:
I think we'd want at least two circuits here, right?

Reply:
```suggestion
    batched_mitigated_executor = mitigate_executor(batched_executor, rule=xx)
    assert np.isclose(*batched_mitigated_executor([circuit_cirq_a] * 3), ddd_value)

```